### PR TITLE
docs: Add issue and pull request templates

### LINKED ISSUE #9: Add Issue template and Pull Request template
This will make it a bit easier to file issues and PRs
https://github.com/devspace/awesome-github-templates

### PR DESCRIPTION
**Pull request by selbekk**
This pull request adds an issue template that asks some basic questions that will help the user write a descriptive issue.

I'm open for discussing the contents - but I think this is one of those things we improve by using it a lot.

Fixes #9

### CODE REVIEW

**Comment by selbekk** [comment]
Not sure if we need a pull request template as well?

**Review by antidecaf — DISMISSED** [review]

**Review by wkillerud — DISMISSED** [review]
A good start :) It may seem odd if filing a question, but we'll work it out as we go.

**Comment by selbekk** [comment]
Not sure how we would deal with usage questions. I can add a comment about wiping the template if you have a question.

**Comment by selbekk** [comment]
I added a pull request template as well. It's not really a template as much as it's a reminder to adhere to CONTRIBUTING.md guidelines.

**Review by wkillerud — DISMISSED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!-- 
+Thanks for opening a pull request! 🎉
+
+If your changes include some kind of UI element, please attach a screenshot of 
+how it looks.
+
+Before submitting a pull request, please have a look through the CONTRIBUTING.md
+document. TL;DR:
+
+- Write proper commit messages tagged with the correct package and scope
```
Comment (antidecaf) [inline PULL_REQUEST_TEMPLATE.md]: Mention conventional commits specifically here, perhaps?

**Review by antidecaf — DISMISSED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (selbekk) [reply PULL_REQUEST_TEMPLATE.md]: Sure. Do you have a suggestion as to how it should be worded?

Comment (antidecaf) [reply PULL_REQUEST_TEMPLATE.md]: > Write proper commit messages tagged with the correct package and scope. We follow the [Conventional Commits](https://conventionalcommits.org/) convention.

... or some such. :)

**Review by henrikhermansen — APPROVED** [review]

**Comment by selbekk** [comment]
@antidecaf ended up with something similar. Unfortunately, this is unparsed markdown, so you can't really have links. Hopefully this will be good enough - at least for a start :)

**Pull request merged**